File: SDKs/amazonaws.com/robomaker/2018-06-29/python/src/sdk/models/shared/simulationjobstatus_enum.py
from dataclasses import dataclass, field
from typing import Enum

class SimulationJobStatusEnum(str, Enum):
    PENDING = "Pending"
    PREPARING = "Preparing"
    RUNNING = "Running"
    RESTARTING = "Restarting"
    COMPLETED = "Completed"
    FAILED = "Failed"
    RUNNING_FAILED = "RunningFailed"
    TERMINATING = "Terminating"
    TERMINATED = "Terminated"
    CANCELED = "Canceled"

File: SDKs/amazonaws.com/robomaker/2018-06-29/go/pkg/models/shared/simulationjobstatusenum.go
package shared




type SimulationJobStatusEnum string

const (
    SimulationJobStatusEnumPending SimulationJobStatusEnum = "Pending"
SimulationJobStatusEnumPreparing SimulationJobStatusEnum = "Preparing"
SimulationJobStatusEnumRunning SimulationJobStatusEnum = "Running"
SimulationJobStatusEnumRestarting SimulationJobStatusEnum = "Restarting"
SimulationJobStatusEnumCompleted SimulationJobStatusEnum = "Completed"
SimulationJobStatusEnumFailed SimulationJobStatusEnum = "Failed"
SimulationJobStatusEnumRunningFailed SimulationJobStatusEnum = "RunningFailed"
SimulationJobStatusEnumTerminating SimulationJobStatusEnum = "Terminating"
SimulationJobStatusEnumTerminated SimulationJobStatusEnum = "Terminated"
SimulationJobStatusEnumCanceled SimulationJobStatusEnum = "Canceled"
)


